_pytadbit/tools/tadbit_parse.py

```diff
@@ -75,9 +83,14 @@ def run(opts):
 
     if not opts.skip:
         logging.info('parsing reads in %s project', name)
-        counts, multis = parse_map(f_names1, f_names2, out_file1=out_file1,
-                                   out_file2=out_file2, re_name=renz, verbose=True,
-                                   genome_seq=genome, compress=opts.compress_input)
+        if opts.mapped1 or opts.mapped2:
```

Comment:
Este if supongo que sobra

Reply:
opts.mapped1 and opts.mapped2 only contain something if you pass some **sam** files as input (ie: `--mapped1 lala_r1.sam --mapped2 lala_r2.sam `). And then you use the `parse_sam` function. Otherwise (standard way), TADbit will search the DB for the **.map** files generated by `tadbit map`, and you than need to use `parse_map`.

Reply:
ok, sorry, my mistake